src/hotspot/share/opto/vectorIntrinsics.cpp

```diff
@@ -510,6 +476,105 @@ bool LibraryCallKit::inline_vector_nary_operation(int n) {
 }
 
 // public static
+// <V extends Vector<E>, E>
+// V unaryOp(long address, Class<? extends V> vClass, Class<E> elementType, int length,
+//           V v,
+//           UnaryOperation<V, ?> defaultImpl)
+//
+// public static
+// <V extends VectorPayload, E>
+// V binaryOp(long address, Class<? extends V> vClass, Class<E> elementType, int length,
+//            V v1, V v2,
+//            BinaryOperation<V, ?> defaultImpl)
```

Comment:
`debugName` parameter is missing

Reply:
It seems the function should be updated to `libraryBinaryOp` and `libraryUnaryOp`?

Reply:
Fixed.